src/networks.json

```diff
@@ -1330,6 +1330,22 @@
     "start": 1683285,
     "logo": "ipfs://QmVh5XRi7xN4UNw7SNKpEhjF1TsPXJnRyNkB4EuMVTVCB6"
   },
+  "9052": {
+    "key": "9052",
+    "name": "Acrechain Mainnet",
+    "shortName": "Acrechain",
+    "chainId": 9052,
+    "network": "mainnet",
+    "multicall": "0x5B614BB4b2939dc4E5EC1399F16c7E78949764CB",
+    "rpc": [
+      "https://evm.acrescan.com"
+    ],
+    "explorer": {
+      "url": "https://acrescout.mindheartsoul.org"
+    },
+    "start": 1690100,
+    "logo": "ipfs://QmbA6CE3G81mm4eDEBUezwcq2qdqEh1zEToG3MbJHirxQt"
```

Comment:
Should pin this file using a provider like pinnata or web3storage etc. right now it is not resolving

Reply:
Can you clarify that you are talking about the logo ipfs file

Reply:
> Should pin this file using a provider like pinnata or web3storage etc. right now it is not resolving

Can you clarify that you are talking about the logo ipfs file

Reply:
yep. Try going here https://ipfs.io/ipfs/QmbA6CE3G81mm4eDEBUezwcq2qdqEh1zEToG3MbJHirxQt should pin the file to pinning service else it won't resolve